.github/workflows/ci.yml

```diff
@@ -103,6 +103,34 @@ jobs:
       - name: Check that guest/rust builds under all configurations
         run: cargo run -p campfire example check-all
 
+  build-web:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: stable
+          target: wasm32-unknown-unknown
+
+      - name: "Build campfire"
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --package campfire
+
+      - name: "Build web client"
+        uses: actions-rs/cargo@v1
+        with:
+          command: run
+          args: --package campfire -- web build --profile release --target standalone
```

Comment:
The `--target standalone` is not necessary.

Removing it will produce a node package (just a package.json added), I kept this here to be consistent and do the exact same parameters as the deploy workflow